recipes/simdutf/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "5.6.2":
+    url: "https://github.com/simdutf/simdutf/archive/v5.6.2.tar.gz"
+    sha256: "325c012e7c29e549779cebfe5a43a3197d9d675b6764f6de547ef19905b56b76"
```

```suggestion
    sha256: "c71b5478c9b912e07f75098f3a60920f1c4de3227b5285ea8a90a2fcf8bd6a89"
```

last version was retagged after @ErniGH reported a missing version bump